index.js

```diff
@@ -0,0 +1,56 @@
+const books = document.getElementById('books');
+const form = document.getElementById('form');
+
+const myLibrary = [];
+
+function displayForm() {
+  if (form.style.display === '') {
+    form.style.display = 'block';
+  } else {
+    form.style.display = '';
+  }
+}
+
+function displaybooks(book) {
+  const li = document.createElement('li');
+  li.className = 'collection-item';
+
+  li.innerHTML = `
+  <h5>Author: <span id="author">${book[0]}</span></h5>
+  <p>Title: ${book[1]} </p>
+  <p>Pages: ${book[2]}</span></p>
+  <a class="delete"><i class="far fa-trash-alt"></i></a>
+  <label>
+  <input type="checkbox" />
+  <span>Read</span>
+</label>
+  `;
+  books.appendChild(li);
+  document.querySelector('#title').value = '';
+  document.querySelector('#author').value = '';
+  document.querySelector('#pages').value = '';
+}
+
```

Comment:
Following the last comment on your review, the display method you implemented did not meet the requirement. 
Here is the last review you received:

```
Kindly create a Display Book method that loops through the array and displays the items in the array.
```